USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -0,0 +1,250 @@
+                    blockText.SetText(textBlock.Text);
+                    blockText.FontSize = 16;
+
+                    if (isBold)
+                    {
+                        blockText.IsBold = true;
+                    }
+                    if (isItalics)
+                    {
+                        blockText.IsItalic = true;
+                    }
+
+                    break;
+                case BDMarker bdMarker:
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker,parentParagraph,true);
+                    }
+                    break;
+                case HMarker hMarker:
+                    XWPFParagraph newHeader = newDoc.CreateParagraph();
+                    XWPFRun headerTitle = newHeader.CreateRun();
+                    headerTitle.SetText(hMarker.HeaderText);
+                    headerTitle.FontSize = 24;
+                    break;
+                case MTMarker mTMarker:
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker);
+                    }
+                    if (!separateChapters)   // No double page breaks before books
+                    {
+                        newDoc.CreateParagraph().CreateRun().AddBreak(BreakType.PAGE);
+                    }
+                    break;
+
+                case FMarker fMarker:
+                    StringBuilder footnote = new StringBuilder();
+
+                    string footnoteId;
+                    switch (fMarker.FootNoteCaller)
+                    {
+                        case "-":
+                            footnoteId = "";
+                            break;
+                        case "+":
+                            footnoteId = $"{FootnoteTextTags.Count + 1}";
+                            break;
+                        default:
+                            footnoteId = fMarker.FootNoteCaller;
+                            break;
+
+                    }
+
+                    XWPFRun footnoteMarker = parentParagraph.CreateRun();
+
+                    footnoteMarker.SetText($"{footnoteId}");
+                    footnoteMarker.Subscript = VerticalAlign.SUBSCRIPT;
+
+                    FootnoteTextTags[footnoteId] = new List<Marker>();
+
+                    foreach (Marker marker in input.Contents)
+                    {
+                        FootnoteTextTags[footnoteId].Add(marker);
+
+                    }
+                    break;
+                case FTMarker fTMarker:
+
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker,parentParagraph);
+                    }
+
+                    break;
+                case FQAMarker fQAMarker:
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker, parentParagraph,false,true);
```

Comment:
Again probably want to use `isItalic: true` instead since that is clearer. And you don't have to specify the first false at all.